plaidml/edsl/tests/BUILD

```diff
@@ -1,18 +1,31 @@
 # Copyright 2020 Intel Corporation
 
 load("@rules_python//python:defs.bzl", "py_test")
-load("//bzl:plaidml.bzl", "plaidml_cc_library", "plaidml_cc_test")
+load("//bzl:plaidml.bzl", "plaidml_cc_binary", "plaidml_cc_library", "plaidml_cc_test")
 load("//pmlc:lit.bzl", "lit_test")
 
 plaidml_cc_test(
     name = "cc_test",
     srcs = ["edsl_test.cc"],
-    tags = ["manual"],  # the `edsl_test.cc.test` rule will run this test via lit
     deps = [
         "//plaidml:testenv",
         "//plaidml/edsl",
         "//plaidml/exec",
+        "//pmlc/rt/vulkan",
```

Comment:
Can we make this conditional on being part of a GPU target?

Reply:
Done